### ISSUE TITLE
Missing Documentation

### ISSUE THREAD

**Issue by FoxxMD** [chromabits/illuminated#1]
This package seems like it could be incredibly useful (for me in particular: routeFactory, apiResponse, queue) but there is no documentation/usage/examples for most of these components.

Do you plan to add more documentation?

**Comment by etcinit** [comment]
Yeah, I intend to add them over time, beginning with modules/components that are sort of stable. I've added the first batch on the commit above ^^^. :)
